pkg/kubectl/cmd/create.go

```diff
@@ -97,11 +97,11 @@ func RunCreate(f *cmdutil.Factory, out io.Writer, filenames util.StringList) err
 	err = r.Visit(func(info *resource.Info) error {
 		data, err := info.Mapping.Codec.Encode(info.Object)
 		if err != nil {
-			return err
+			return fmt.Errorf("error when creating %q: %v ", info.Source, err)
```

Comment:
Shouldn't be necessary. The error returned here gets handled in [`CheckErr`](https://github.com/GoogleCloudPlatform/kubernetes/pull/10221/files#diff-a0580db929d5a6aeb59e7473a86017adR54).